.travis.yml

```diff
@@ -39,33 +39,34 @@ jobs:
       after_success:
       - codecov
 
-    - stage: e2e
+    - stage: "Test"
       name: "docker - calico e2e Tests"
       language: python
       cache: pip
       python: "3.8"
       env:
       - KUBECONFIG="$HOME/.kube/config"
-      # TODO upgrade to v1.18.6 or newer
-      - KUBERNETES_VERSION=v1.15.3
-      # TODO update to v1.12.1 or newer
-      - MINIKUBE_VERSION=v1.3.1
+      # stable version in minikube v1.12.1
+      - KUBERNETES_VERSION=v1.18.3
+      - MINIKUBE_VERSION=v1.12.1
```

Comment:
Could you directly use `v1.12.2`? There are no breaking changes and everythinng should just work.